### DISCUSSION TITLE
Change Tags Index Header Size

### DISCUSSION THREAD

**Discussion by GL513** [squidfunk/mkdocs-material#5174] [Q&A] [answered]
Hello again.
I was messing with tags and noticed that the header sizes for the tags index are too small for my liking. How would I go about changing the size of the headers?

**Comment by squidfunk** [comment, marked as answer]
You can use [additional CSS](https://squidfunk.github.io/mkdocs-material/customization/#additional-css) and specifically target the tags in headers, maybe something like (untested, on my mobile right now):

``` css
.md-typeset h2 .md-tag {
  font-size: 20px;
}
```

**Reply by GL513** [reply]
Yep that took care of it, ha, thank you again!

**Reply by squidfunk** [reply]
Awesome, I love when "blind coding" works 😂
